src/main/java/intellistart/interviewplanning/controllers/dto/FacebookOauthInfoDto.kt

```diff
@@ -1,10 +1,10 @@
 package intellistart.interviewplanning.controllers.dto
 
-/**
- * DTO for facebook client id.
- */
 data class FacebookOauthInfoDto(
-    var clientId: String,
-    var redirectUri: String,
-    var tokenRequestUrl: String
+
+    val clientId: String,
```

Comment:
Here you dont initialize. Try to stick to one style, and not a different one in each file